data_generators/job_description_scraper.py

```diff
@@ -524,7 +526,7 @@ def remove_job_titles(self, job_list):
                 job_list.append(parse.quote(job))
             self.search_terms = [
                 x for x in self.search_terms if x
-                not in job_list
+                                                not in job_list
```

Comment:
This change wasn't needed, revert and it should be good to go 👍

Reply:
Ah cats adding white space. It's been removed, fits inline with original indent.